src/pages/IndividualArticle.jsx

```diff
@@ -20,16 +22,37 @@ export default function IndividualArticle () {
                 console.log(err)
             }
         }
+
+        const getComments = async () => {
+            try {
+                const response = await newsApi.get(`/${article_id}/comments`)
```

Comment:
if you're extracting the API calls away into their own file this becomes one function call e.g. `await getComments(article_id)`